se_module/sybase.te

```diff
@@ -0,0 +1,266 @@
+policy_module(sybase, 0.1.0)
+
+########################################
+#
+# Declarations
+#
+########################################
+
+attribute sybase_db_domain;
+
+attribute sybase_db_type;
+attribute sybase_file_type;
+attribute sybase_exec_type;
+attribute sybase_port_type;
+
+
+type sybase_t;
+application_type(sybase_t)
+typeattribute sybase_t sybase_db_domain;
+
+type  sybase_exec_t;
+application_executable_file(sybase_exec_t)
+typeattribute sybase_exec_t sybase_exec_type;
+typeattribute sybase_exec_t sybase_file_type;
+
+type sybase_bin_t;
+corecmd_executable_file(sybase_bin_t)
+typeattribute sybase_bin_t sybase_file_type;
+
+type sybase_lib_t;
+files_type(sybase_lib_t)
+typeattribute sybase_lib_t sybase_file_type;
+
+type sybase_conf_t;
+logging_log_file(sybase_conf_t)
+typeattribute sybase_conf_t sybase_file_type;
+
+type sybase_var_t;
+files_type(sybase_var_t)
+typeattribute sybase_var_t sybase_file_type;
+
+type sybase_run_t;
+files_type(sybase_run_t)
+typeattribute sybase_run_t sybase_file_type;
+
+type sybase_tmp_t;
+files_tmp_file(sybase_tmp_t)
+typeattribute sybase_tmp_t sybase_file_type;
+
+type sybase_log_t;
+files_config_file(sybase_log_t)
+typeattribute sybase_log_t sybase_file_type;
+
+type sybase_db_t;
+files_type(sybase_db_t)
+typeattribute sybase_db_t sybase_db_type;
+
+type            sybase_unit_file_t;
+systemd_unit_file(sybase_unit_file_t);
+
+type  sybase_port_t;
+corenet_port(sybase_port_t)
+typeattribute sybase_port_t sybase_port_type;
+
+########################################
+#
+# Requires
+#
+########################################
+
+gen_require(`
+    attribute unconfined_domain_type;
+    attribute initrc_transition_domain;
+    attribute confined_admindomain;
+
+    type      unconfined_t;
+    type      unconfined_service_t;
+    type      initrc_t;
+    type      init_t;
+    
+    type      sysadm_t;
+    type      dbadm_t;
+
+    type      tmpfs_t;
+
+    role      unconfined_r;
+    role      system_r;
+')
+
+########################################
+#
+# Application domain definition
+#
+########################################
+
+init_daemon_domain(sybase_t, sybase_exec_t)
+
+role  system_r  types sybase_db_domain;
```

Comment:
That would allow the `dbadm_r` to run the Sybase ASE processes, wouldn't it ?

We don't want the DBAs to run the DB engines as if they were the DB engine...